### DISCUSSION TITLE
Create the AWS WAF table to use Amazon Athena to query WAF logs

### DISCUSSION THREAD

**Discussion by mayakodivila** [aws/aws-cdk#25769] [Q&A]
I am looking forward to using AWS athena to query WAF logs stored in S3 bucket. Is it possible to create the WAF glue tables using AWS CDK or it is better to create them as a NamedQuery?

**Comment by yordaa-me** [comment]
Im able to create a glue catalogue through terraform which uses the AWS SDK underneath so I don't see why you wouldn't be able to with CDK
